build.sbt

```diff
@@ -0,0 +1,85 @@
+import Dependencies.appDependencies
+import play.sbt.routes.RoutesKeys.routesImport
+import sbt.Tests.{ Group, SubProcess }
+import uk.gov.hmrc.DefaultBuildSettings._
+import uk.gov.hmrc.SbtAutoBuildPlugin
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+import uk.gov.hmrc.versioning.SbtGitVersioning
+import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
+
+lazy val plugins: Seq[Plugins] = Seq.empty
+lazy val playSettings: Seq[Setting[_]] = Seq.empty
+
+organization := "uk.gov.hmrc"
+name := "gform-frontend"
+
+coverageEnabled := true
+
+lazy val scoverageSettings = {
+  import scoverage.ScoverageKeys
+  Seq(
+    // Semicolon-separated list of regexs matching classes to exclude
+    ScoverageKeys.coverageExcludedPackages := """uk.gov.hmrc.BuildInfo;._.Routes;._.RoutesPrefix;._Filters?;MicroserviceAuditConnector;Module;GraphiteStartUp;._.Reverse[^.]*""",
+    ScoverageKeys.coverageMinimum := 80.00,
+    ScoverageKeys.coverageFailOnMinimum := false,
+    ScoverageKeys.coverageHighlighting := true,
+    parallelExecution in Test := false
+  )
+}
+
+enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory)
+majorVersion := 0
+PlayKeys.playDefaultPort := 9195
+playSettings
+scalaSettings
+publishingSettings
+defaultSettings()
+
+scalafmtOnCompile := true
+scalaVersion := "2.11.12"
+libraryDependencies ++= appDependencies
+evictionWarningOptions in update := EvictionWarningOptions.default.withWarnScalaVersionEviction(false)
+routesImport ++= Seq(
+  "uk.gov.hmrc.gform.binders.ValueClassBinder._",
+  "uk.gov.hmrc.gform.sharedmodel.formtemplate._",
+  "uk.gov.hmrc.gform.sharedmodel.form._",
+  "uk.gov.hmrc.gform.sharedmodel._",
+  "uk.gov.hmrc.gform.auth._",
+  "uk.gov.hmrc.gform.models._"
+)
+//TODO enable it and fix errors in code!!!
```

Comment:
what errors? is this still applicable? perhaps we can remove this comment? see if this is still required, if is see what the errors are about and add some key words from the logs

Reply:
Yes, this is still applicable. If you wonder what errors, do as comments ask and see for yourself :)

Reply:
To enable this by default we would need first to get rid of all warnings we have now, but that is not so simple.